configurate-core/src/test/java/ninja/leaping/configurate/objectmapping/TypeSerializersTest.java

```diff
@@ -264,12 +293,70 @@ public void testAnnotatedObjectSerializer() throws ObjectMappingException {
         final ConfigurationNode node = SimpleConfigurationNode.root();
         node.getNode("int").setValue("42");
         node.getNode("name").setValue("Bob");
-
+        List<Integer> list = Arrays.asList(72, 101, 108, 108, 111, 32, 119, 111, 114, 108, 100);
+        node.getNode("list").setValue(list);
         TestObject object = testObjectSerializer.deserialize(testNodeType, node);
         assertEquals(42, object.value);
         assertEquals("Bob", object.name);
+        assertEquals("Hello world", object.list);
+
+        ConfigurationNode node2 = SimpleConfigurationNode.root();
+        testObjectSerializer.serialize(TypeToken.of(TestObject.class), object, node2);
+        assertEquals(42, node2.getNode("int").getValue());
+        assertEquals("Bob", node2.getNode("name").getValue());
+        List<Integer> list2 = (List) node2.getNode("list").getValue();
+
+        assertTrue(Arrays.equals(list.toArray(), list2.toArray()));
     }
 
+    @ConfigSerializable
+    private static class TestObject2 {
+        @Setting @Adapter(NoDefCtrTester.class) private String field;
+    }
+
```

Comment:
All of the code from this point onwards has been indented with tabs - should be 4 spaces.

Might not be obvious in your IDE, but if you view on GitHub it's super obvious. :)